src/main/java/uk/gov/register/views/v2/BlobView.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.register.views.v2;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import uk.gov.register.core.Field;
+import uk.gov.register.core.FieldValue;
+import uk.gov.register.core.Item;
+import uk.gov.register.core.StringValue;
+import uk.gov.register.service.ItemConverter;
+import uk.gov.register.views.CsvRepresentationView;
+import uk.gov.register.views.representations.CsvRepresentation;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class BlobView implements CsvRepresentationView<Map<String, FieldValue>> {
+    private final Iterable<Field> fields;
+    private final Map<String, FieldValue> fieldValueMap;
+
+    public BlobView(Item item, final Map<String, Field> fieldsByName, final ItemConverter itemConverter) {
+        this.fields = fieldsByName.values();
+        this.fieldValueMap = new HashMap(itemConverter.convertItem(item, fieldsByName));
```

Comment:
If you change this to `new HashMap<>` it fixes an IntelliJ warning about unchecked assignment.

Reply:
I wonder if we should use a `LinkedHashMap` here to preserve the order, else I *think* we risk an inconsistently ordered response? Not that we make any guarantees about this but it is good to be consistent for caching etc.